js/repl/TimeTravelSlider.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+
+import { css } from "emotion";
+import React from "react";
+import { colors } from "./styles";
+
+type Props = {
+  className?: string,
+  currentTransition: Object,
+  transitions: Array<Object>,
+  selectTransition: (transition: Object) => () => void,
+};
+
+class TimeTravelSlider extends React.Component<Props> {
+  _isMounted = false;
+  componentDidMount = () => {
+    this._isMounted = true;
+  };
+
+  componentWillUnmount = () => {
+    this._isMounted = false;
+  };
+
+  render() {
+    const { transitions, currentTransition } = this.props;
+    return (
+      <div className={styles.sliderWrapper}>
+        <div className={styles.sliderRow}>
+          {transitions &&
```

Comment:
Nit: Wondering if we need to check if transitions is truthy, since the `Props` above state that it can't?

Reply:
@existentialism sounds right, removed checking